### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by Batsenko**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html lang="en" class="html">
+<head>
+    <meta charset="UTF-8">
+    <script src="main.js"></script>
+</head>
+<body>
+</body>
+</html>
```
Comment (gestap) [inline index.html]: the document must end with one empty line

File: main.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        
+        return Math.min.apply(null, this.intArray);
+    }
+
+    max() {
+        if(this.intArray.length === 0) throw "exception";
+    
+        return Math.max.apply(null, this.intArray);
+    }
+}
```
Comment (gestap) [inline main.js]: the document must end with one empty line

File: main.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
+class Numberdrome {
+    constructor() {
```
Comment (gestap) [inline main.js]: use 2 whitespaces instead of 4

File: main.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
+class Numberdrome {
```
Comment (gestap) [inline main.js]: use strict mode

**Review by ost-ptk — CHANGES_REQUESTED** [review]

**Review by ost-ptk — APPROVED** [review]

**Pull request closed without merging**